os_net_config/utils.py

```diff
@@ -927,3 +927,12 @@ def ethtool_path():
         logger.warning("Could not execute /sbin/ethtool or /usr/sbin/ethtool")
         return False
     return ethtoolcmd
+
+
+def set_accept_ra_sysctl(iface):
+    """Set /proc/sys/net/ipv6/conf/<iface>/accept_ra """
+    filename = "/proc/sys/net/ipv6/conf/{}/accept_ra".format(iface)
+    if os.path.exists(filename):
+        logger.info("DEFROUTE=no: Set sysctl to ignore router advertisements")
+        with open(filename, 'w') as f:
+            f.write("0")
```

Comment:
is persistence across reboots needed ? What if there is a need to change this value to 1 later ?

Reply:
I added a comment. This sets the sysctl immediately if the interface isn't restarted, but after that the ifcfg settings will cause the ifup script to set the sysctls.